frontend/src/views/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,50 @@
+import './HomePage.scss';
+import 'typeface-roboto';
+
+import Button from '@mui/material/Button';
+import React from 'react';
+
+import buttoStyles from '../../components/Button/button.module.scss';
+import Calendar from '../../components/Calendar/Calendar';
+import Clock from '../../components/Clock/Clock';
+import Date from '../../components/Date/Date';
+
+function homePage() {
+  return (
+    <div className="homeContainer">
+      <div className="textContetn">
+        <div className="title">
+          <h1>aID kIT</h1>
+          <h2>Your medical assistant</h2>
+        </div>
+        <ul className="navList">
+          <Button variant="contained" className={buttoStyles.Button}>
+            DAILY DRUGS
+          </Button>
+          <br />
+          <Button variant="contained" className={buttoStyles.Button}>
+            CREDITS
+          </Button>
+          <br />
+          <Button variant="contained" className={buttoStyles.Button}>
+            LEXICON
+          </Button>
+          <br />
+          <Button variant="contained" className={buttoStyles.Button}>
+            SETTINGS
+          </Button>
```

Comment:
Each Button here looks the same. Consider having it defined in the map of objects eg. [{buttonLabel : DAILY DRUGS }, {buttonLabel : CREDITS} etc.]. Then use a .map method in the return function.